hBank/src/main/java/animalPark/AnimalInterface.java

```diff
@@ -0,0 +1,5 @@
+package animalPark;
+
+public interface AnimalInterface {
```

Comment:
Interface names should be adjectives. They should end with “able” or “ible” whenever the interface provides a capability; otherwise, they should be nouns. Interface names follow the same capitalization convention as class names: public interface Serializable {...} public interface SystemPanel {...}

For this case I would suggest to use something like "Noisy", or "Soundful"

Reply:
Done